variables.tf

```diff
@@ -463,3 +463,39 @@ variable "graphdb_enable_userdata_scripts_on_reboot" {
   type        = bool
   default     = false
 }
+
+#cmk building in progress
```

Comment:
Leave empty space between the CMK and the #. Also CMK should be with capital letters.

Reply:
No, thank you for noticing. I left that one for me to add more variables later.